src/models/cookSchema.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const cookSchema = mongoose.Schema({
+  cook_address: {
+    type: String,
+    required: true,
+  },
+  cook_phone: {
+    type: String,
+    required: true,
+  },
+  cook_info: {
+    type: String,
+  },
+  cook_name: {
+    type: String,
+  },
```

Comment:
I think it is a good idea we have created_at attribute here in order to find out when it is created (Most probably I forgot to put in the design).